README.md

```diff
@@ -20,17 +20,18 @@ Error: Resource not accessible by integration
 
 ## Inputs
 
-| input                      | required | default                  | description                                                     |
```

Comment:
This diff looks a bit large because the table had to be centered to keep aliment. 

To aid in review, I can confirm that the only change is the last entry of the table. 

You can render it here : https://github.com/yeikel/github-action-merge-dependabot/blob/feat/skip-verification/README.md